backend/src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+#RabbitMQ Server configuration
+spring:
+  rabbitmq:
+    host: javaschoolrmqrv2dutokzu7as-vm0.southcentralus.cloudapp.azure.com
+    port: 5672
+    username: user
+    password: JavaSchool.1234
+    template:
+      default-receive-queue: db-queue
+      exchange: db-exchange
+      routing-key: "#"
+
+
+config-queue:
+  name: db-queue
```

Comment:
Why do you have the name of the queue if you only need the exchange and routing key?
`ShippingRequestSenderImpl.java#sendRequest`

Reply:
Just addit to the Rabbit configuration the set the default receptionqueue